packages/cli/src/commands/spaces/info.ts

```diff
@@ -32,15 +34,18 @@ export default class Info extends Command {
       `))
     }
 
-    let headers = {}
+    const headers: IncomingHttpHeaders = {
+      Accept: 'application/vnd.heroku+json; version=3.fir',
+    }
     if (!flags.json) {
-      headers = {'Accept-Expansion': 'region'}
+      headers['Accept-Expansion'] = 'region'
     }
 
-    const {body: space} = await this.heroku.get<Heroku.Space>(`/spaces/${spaceName}`, {headers})
+    const {body: updatedSpace} = await this.heroku.get<Space>(`/spaces/${spaceName}`, {headers})
+    const space: SpaceWithOutboundIps = updatedSpace
```

Comment:
Rather than setting the body's `updatedSpace` variable as `Space` type and then immediately setting it as `SpaceWithOutboundIps` type, couldn't we just set it as `SpaceWithOutboundIps`? So like:

```javascript
const {body: space} = await this.heroku.get<SpaceWithOutboundIps>(`/spaces/${spaceName}`, {headers})

Reply:
Good thought, thank you! Updated.